uber/models/attendee.py

```diff
@@ -749,6 +749,12 @@ def is_not_ready_to_checkin(self):
     def can_abandon_badge(self):
         return not self.amount_paid and not self.paid == c.NEED_NOT_PAY and not self.is_group_leader
 
+    @property
+    def can_refund_badge(self):
+        return self.amount_paid and self.paid not in [c.NEED_NOT_PAY, c.REFUNDED] \
```

Comment:
Should `not self.is_group_leader` be `not self.group_id`? I actually don't know!

`not self.is_group_leader` is preventing a refund from being given to a group leader.

`not self.group_id` would prevent a refund from being given to _anyone_ in a group.

Reply:
I think we should let people refund their group badge. This would process a refund for anyone who's kicked in extra. Group members can currently already abandon their own badges with the "I'm not coming" button.

The group leader can't get a refund because we'd be invalidating the entire group's badges. 😅

Reply:
So if a group member refunds their badge, the refund actually goes to the group leader, right? And the group's badge count is decremented by one?

Or is this just refunding the extra kickin and releasing the badge back to the group?

Reply:
This is just refunding the extra kickin and releasing the badge back to the group. :)